test/e2e/crud.go

```diff
@@ -92,68 +91,76 @@ var _ = Describe("Federated types", func() {
 
 			// TODO(marun) Is there a better way to create crd's from code?
 
-			crdKind := "FedTestCrd"
+			targetCrdKind := "FedTestCrd"
+			targetCrd := newTestCrd(tl, targetCrdKind)
```

Comment:
The goal of this test was to validate federation of a CRD target, since all the existing CRUD tests target first-class api types.  I didn't test overrides because we are already validating CRD-based overrides in most of the CRUD tests.  What benefit(s) do you see in adding a check for overrides to this test?

Reply:
Validating FederatedTypeConfig with crd overrides is already being done when e2e crud is targeting unmanaged, though. There is no difference between a FederatedTypeConfig loaded from config and one a user would define (since fed primitives are CRD in both cases).  This test validates that the sync controller is capable of working with a CRD as the target, which is a corner case the other crud tests don't cover.